lua/refactoring/tests/lsp_utils_spec.lua

```diff
@@ -58,7 +58,7 @@ describe("lsp_utils", function()
         local region = Region:from_current_selection()
         local insert_text = lsp_utils.insert_text(region, "hello, piq")
         local bufnr = vim.api.nvim_get_current_buf()
-        vim.lsp.util.apply_text_edits({ insert_text }, bufnr)
+        vim.lsp.util.apply_text_edits({ insert_text }, bufnr, "utf-16")
```

Comment:
is it always utf-16?

Reply:
It usually is, but it might not be. It can be fetched from the LSP client with something like 
```lua
vim.lsp.buf_get_clients(bufnr)[1].offset_encoding
```

Reply:
Yeah I like that idea better, I'll change it - thanks @zemzale.

Reply:
On second thought, it seems that this won't work because our test system does not account for LSP - so we'll keep it as `utf-16` for now to fix the tests and later perhaps we'll find a way to incorporate LSP into testing